capture/runbook.go

```diff
@@ -449,6 +453,18 @@ func (c *cRunbook) CaptureExecStdout(stdout string) {
 	r.currentExecTestCond = append(r.currentExecTestCond, fmt.Sprintf("current.stdout == %#v", stdout))
```

Comment:
If we integrate CaptureExecStdout and CaptureExecStdoutLine, what should we do here?